EIPS/eip-5521.md

```diff
@@ -56,73 +58,14 @@ This standard can be fully [ERC-721](./eip-721.md) compatible by adding an exten
 
 ## Test Cases
 
-Truffle and Openzeppelin are required to run the following in a test network.
-
-```node
-
-truffle develop
-
-rNFT = await ERC_rNFT.new("ERC_5521", "ERC_5521")
-rNFT.safeMint(1, [], [])
-rNFT.referredOf(1)
-rNFT.referringOf(1)
-
-rNFT.safeMint(2, [rNFT.address], [[1]])
-rNFT.referredOf(2)
-rNFT.referringOf(2)
-
-rNFT.safeMint(3, [rNFT.address_1, rNFT.address_2], [[1,2], [2,4,5]])
-rNFT.referredOf(2)
-rNFT.referredOf(3)
-rNFT.referringOf(3)
-
-```
+Test cases are included in [ERC_5521.test.js](../assets/eip-5521/ERC_5521.test.js)
 
 ## Reference Implementation
 
 ```solidity
 
 // SPDX-License-Identifier: MIT
```

Comment:
Code included in the EIP itself _must_ be CC0-1.0 licensed.

Reply:
> Code included in the EIP itself _must_ be CC0-1.0 licensed.

Not sure if this is the right way to do though, I removed the `SPDX-License-Identifier: MIT`.

Reply:
> Code included in the EIP itself _must_ be CC0-1.0 licensed.

Not sure if this is the right way to do though, I removed the SPDX-License-Identifier: MIT.